pkg/pillar/volumehandlers/handlers.go

```diff
@@ -64,7 +64,7 @@ func GetVolumeHandler(log *base.LogObject, volumeManager VolumeMgr, status *type
 	common := commonVolumeHandler{volumeManager: volumeManager, status: status, log: log}
 
 	if base.IsHVTypeKube() {
-		return &volumeHandlerCSI{commonVolumeHandler: common, useVHost: false} // kubevirt does not support vhost yet
+		return NewCSIHandler(common, false) // kubevirt does not support vhost yet
```

Comment:
Isn't this GetVolumeHandler() called by all? or if in kubevirt mode, this handler is not invoked?

Reply:
ok, got it, we include different directories in compiling